README.md

```diff
@@ -130,6 +130,14 @@ If using Ax in Jupyter notebooks:
 pip3 install git+ssh://git@github.com/facebook/Ax.git#egg=Ax[notebook]
 ```
 
+To support plotly-based plotting in newer Jupyer notebook versions 
+
+```
+pip install "notebook>=5.3" "ipywidgets==7.5"
```

Comment:
Should we also mention that Plotly >= 4.0?

Reply:
Is it strictly necessary? I didn't think we were compatible